src/rustdoc.md

```diff
@@ -107,21 +107,26 @@ deprecate that][44136]. If you need finer-grain control over these passes, pleas
 
 Here is current (as of this writing) list of passes:
 
-- `collapse-docs` is necessary because each line of a doc comment is given as a
+- `propagate-doc-cfg` - propagates `#[doc(cfg(...))]` to child items.
+- `collapse-docs` concatenates all document attributes into one document
+  attribute. This is necessary because each line of a doc comment is given as a
   separate doc attribute, and this will combine them into a single string with
   line breaks between each attribute.
-- `unindent-comments` is necessary because the convention for writing
+- `unindent-comments` removes excess indentation on comments in order for
+  markdown to like it. This is necessary because the convention for writing
   documentation is to provide a space between the `///` or `//!` marker and the
   text, and stripping that leading space will make the text easier to parse by
   the Markdown parser. (In my experience it's less necessary now that we have a
   Commonmark-compliant parser, since it doesn't have a problem with headers
   that have a space before the `##` that marks the heading.)
```

Comment:
I'm not sure this parenthetical is worth keeping? It's definitely not adding as-is, since the rest of the document has so far avoided first-person. Maybe having a historical note that the previous Markdown parser had a problem with spaces before a heading would be useful, but i'm less confident about the remark about it being "less necessary" since i'm not sure that Pulldown is totally okay with everything having an indent on it.

Reply:
Better?